qa/distros/all/centos_9.stream.yaml

```diff
@@ -7,3 +7,6 @@ overrides:
   install:
     ceph:
       enable_coprs: [ceph/el9]
+      enable_coprs: [aflyhorse/iozone]
```

Comment:
Just wondering why this is specific to CentOS Stream 9.  Where is `iozone` coming from on CentOS Stream 8?

Reply:
I installed a centos-stream 8 and have a test we also need to do the same fix. It seems we haven't hit the failure on centos-stream 8 yet ?

Reply:
It seems before our run were on mainly on centos 8/9 instead of centos-stream ?

Reply:
> > It seems we haven't hit the failure on centos-stream 8 yet ?
> 
> Need to figure out why -- `iozone` might already be packaged somewhere else.

https://pulpito.ceph.com/yuriw-2023-07-17_15:50:05-fs-wip-yuri3-testing-2023-07-14-0724-quincy-distro-default-smithi/7341898/

Please see this one, which succeeded. The description said it want the `centos0` disto, but finally it switched to `rhel8` instead.

Reply:
> Please see this one, which succeeded. The description said it want the `centos0` disto, but finally it switched to `rhel8` instead.

That's funny, why would that happen o_O

Reply:
> > Please see this one, which succeeded. The description said it want the `centos8` disto, but finally it switched to `rhel8` instead.
> 
> That's funny, why would that happen o_O

`overrides/{distro/stock/{k-stock rhel_8}` It was overrode.

https://pulpito.ceph.com/yuriw-2023-07-14_23:45:56-fs-reef-distro-default-smithi/7338917/

This one was using the `centos-stream 8` and it succeeded, but I still couldn't find where installed the iozone package. And also check the centos-stream 8 repo mirror, also couldn't find the package.

So strange :-(

Reply:
> > > Please see this one, which succeeded. The description said it want the `centos8` disto, but finally it switched to `rhel8` instead.
> > 
> > 
> > That's funny, why would that happen o_O
> 
> `overrides/{distro/stock/{k-stock rhel_8}` It was overrode.

Ah, that's expected then.

> 
> https://pulpito.ceph.com/yuriw-2023-07-14_23:45:56-fs-reef-distro-default-smithi/7338917/
> 
> This one was using the `centos-stream 8` and it succeeded, but I still couldn't find where installed the iozone package. And also check the centos-stream 8 repo mirror, also couldn't find the package.
> 
> So strange :-(

Probably a part of Base OS? Can't tell...

Reply:
> Probably a part of Base OS? Can't tell...

I think this needs to be determined before we add any Copr repos.  If `iozone` is coming from _somewhere_ for CentOS Stream 8 and RHEL 8, chances are it could come from the same place in for CentOS Stream 9.

Reply:
In theory the iozone shouldn't be in the Base OS from my understanding, because it's license was clarified as [non-free shareware](https://lists.fedoraproject.org/archives/list/epel-announce@lists.fedoraproject.org/thread/C47ZSUGBFVDCMYQK3INTIY7C7TXJSDOA/) and it was removed from both Fedora and EPEL.

Reply:
> > Probably a part of Base OS? Can't tell...
> 
> I think this needs to be determined before we add any Copr repos. If `iozone` is coming from _somewhere_ for CentOS Stream 8 and RHEL 8, chances are it could come from the same place for CentOS Stream 9.

From the `teuthology.log` I couldn't find any log about installing the `iozone` package. Any idea to confirm this ?

Reply:
> From the `teuthology.log` I couldn't find any log about installing the `iozone` package. Any idea to confirm this ?

Ah, I think it's embedded in the fog image.  We have lab-extras repo for CentOS (Stream) 8, see https://github.com/ceph/ceph-cm-ansible/blob/01127c9de36b64f86911fcaf879081994f8f417f/roles/testnode/vars/centos_8.yml#L5-L10.  For CentOS Stream 9, it doesn't seem to exist and `iozone` just got commented out: https://github.com/ceph/ceph-cm-ansible/blob/01127c9de36b64f86911fcaf879081994f8f417f/roles/testnode/vars/centos_9.yml#L76.

@dmick Can lab-extras repo be resurrected for CentOS Stream 9?  If not, I would suggest adding this Copr repo to https://github.com/ceph/ceph-cm-ansible/blob/main/roles/testnode/vars/centos_9.yml.  Having jobs install `iozone` on their own each time as proposed here is suboptimal.

Reply:
I don't know the history or ownership of lab-extras, but it seems sensible on its surface that it will continue to be needed.  Who's worked on it in the past?

Reply:
> Who's worked on it in the past?

David, I think.

Reply:
the `lab-extra` hasn't support the `9` yet.